sections/dom.include

```diff
@@ -604,140 +604,75 @@
+      &lt;p&gt;A trilogy of five books by Douglas Adams.&lt;/p&gt;
+    &lt;/main&gt;
+  &lt;/body&gt;
+&lt;/html&gt;
     </pre>
-
-    Because HTML conveys <em>meaning</em>, rather than presentation, the same page can also be used
-    by a small browser on a mobile phone, without any change to the page. Instead of headings being
-    in large letters as on the desktop, for example, the browser on the mobile phone might use the
-    same size text for the whole the page, but with the headings in bold.
-
-    But it goes further than just differences in screen size: the same page could equally be used by
-    a blind user using a browser based around speech synthesis, which instead of displaying the page
-    on a screen, reads the page to the user, e.g., using headphones. Instead of large text for the
-    headings, the speech browser might use a different volume or a slower voice.
-
-    That's not all, either. Since the browsers know which parts of the page are the headings, they
-    can create a document outline that the user can use to quickly navigate around the document,
-    using keys for "jump to next heading" or "jump to previous heading". Such features are
-    especially common with speech browsers, where users would otherwise find quickly navigating a
-    page quite difficult.
-
-    Even beyond browsers, software can make use of this information. Search engines can use the
-    headings to more effectively index a page, or to provide quick links to subsections of the page
-    from their results. Tools can use the headings to create a table of contents (that is in fact
-    how this very specification's table of contents is generated).
-
-    This example has focused on headings, but the same principle applies to all of the semantics
-    in HTML.
-  </div>
-
-  Authors must not use elements, attributes, or attribute values for purposes other than their
-  appropriate intended semantic purpose, as doing so prevents software from correctly processing the
-  page.
+</div>
+
+This semantic information is critical to assistive technologies. For example, a screen reader will query the browser for semantic information and use that information to present the document or application in synthetic speech.
+
+In some cases assistive technologies use semantic information to provide additional functionality. A speech recognition tool might provide a voice command for moving focus to the start of the main element for example.
+
+When the appropriate HTML element or attribute is not used, it deprives HTML processors of valuable semantic information.
+
+<div class="example">
+<p>In this example the visual manifestation of the document may look the part, but because the appropriate HTML elements have not been used there is little semantic information available to browsers, search engines or assistive technologies.</p>
+
+<pre highlight="html">
+    &lt;!doctype html&gt;
+&lt;html lang="en"&gt;
```

Comment:
Indentation. Note the doctype is indented too far. It should only be 2 space in fro the pre tag.